frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Avatar as ChakraAvatar,
+  AvatarBadge as ChakraAvatarBadge,
+  AvatarProps as ChakraAvatarProps,
+  Box,
+  ButtonProps,
+  Icon,
+  MenuItemProps,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+import { BxsUser } from '~/assets/icons/BxsUser'
+
+import DropdownMenu from '../Menu'
+
+export interface AvatarProps extends ChakraAvatarProps {
+  hasNotification?: boolean
+}
+
+/*
+ * Utility to extract first letter of avatar name
+ * Override default getInitials Avatar method
+ */
+const getInitials = (fullName: string) => fullName[0]
```

Comment:
nit: should this be inlined into the Avatar component (and only used if props.getInitials are not provided?

Reply:
edited